### ISSUE TITLE
Web documentation 📖

### ISSUE THREAD

**Issue by paualarco** [monix/monix-kafka#202]
Looking at how nice and organised the documentation of other monix projects (bio and connect) looks like, thought that it could be a good idea to migrate kafka documentation on the web too.  

And while migrating it, there are some points that could be added as well such like producer and consumer configurations, how different ser/de are handled in monix-kafka, avro schema registry and finally some tips and instructions for local testing (embedded vs docker).

**Comment by Avasil** [comment]
Yes, the main issue is that someone needs to write it and it's very low on my personal priorities atm. Writing docs for `monix-bio` is already draining me :D

**Comment by paualarco** [comment]
If you thing it is a good idea, I can just do it. 
For the moment would just aim to migrate the actual documentation on the web, and later on to add the possible new content as mentioned in the description?  
That could include migrating to github actions and automated releases too.

**Comment by Avasil** [comment]
If you're interested in it then of course, it would be great!

**Comment by alexandru** [comment]
@paualarco would be great ❤️

Btw, I can help out with providing sub-domains of `monix.io`. @Avasil if you want I can provide access to the DNS settings on Cloudflare. 

Not sure if you prefer for the docs to remain on https://monix.github.io/monix-bio/ or if you'd like `bio.monix.io`.

**Comment by Avasil** [comment]
> Not sure if you prefer for the docs to remain on https://monix.github.io/monix-bio/ or if you'd like bio.monix.io.

It's not important to me right now but I'd like to have everything under one domain eventually (perhaps in a common design?). 

I was thinking about `https://monix.io/docs/monix-bio`, `https://monix.io/docs/monix-connect` etc. Could be without `monix` prefix but I'm not sure how to call `monix/monix` (probably something like `core`) and if it's hard to setup common docs in multirepo

**Comment by paualarco** [comment]
> It's not important to me right now but I'd like to have everything under one domain eventually (perhaps in a common design?).

That would be perfect :) 
Having a separate project to abstract the generic web configuration would be ideal.
